tcpdump.c

```diff
@@ -1273,6 +1277,21 @@ main(int argc, char **argv)
 			++pflag;
 			break;
 
+		case OPTION_PRINT_BYTES:
+			if ((colon=strchr(optarg,':')) == 0) {
```

Comment:
maybe { char *colon; ...} so that we don't need another function-wide automatic.